python/cudf/cudf_pandas_tests/test_cudf_pandas.py

```diff
@@ -1421,3 +1421,24 @@ def test_holidays_within_dates(holiday, start, expected):
             utc.localize(xpd.Timestamp(start)),
         )
     ) == [utc.localize(dt) for dt in expected]
+
+
+def test_pandas_debugging_mode_option(monkeypatch):
```

Comment:
We have a test for the warning when results are different. Can we add similar tests for:
- The result from pandas could not be computed.
  - Try mocking a method in pandas to raise an error (must be a method that supports the cudf fast path).
- Pandas debugging mode failed.
  - Try mocking a method in cudf to return something that cannot be compared with `assert_eq`, so that the assertion code fails (perhaps returning a `None` value would be enough?).